cfme/web_ui/__init__.py

```diff
@@ -3056,6 +3056,10 @@ class DynamicTable(Pretty):
     """
     pretty_attrs = "root_loc", "default_row_item"
     ROWS = ".//tbody/tr[not(contains(@id, 'new_tr'))]"
+    DELETE_ALL = {
```

Comment:
Should it be called ```DELETE_ALL```? Doesn't it delete a single field? Is ```ALL``` referencing all versions?